content/en/docs/tutorials/configuration/updating-configuration-via-a-configmap.md

```diff
@@ -0,0 +1,322 @@
+---
+title: Updating configuration via a ConfigMap
+content_type: tutorial
+---
+
+<!-- overview -->
+
+This page provides a step-by-step example updating configuration within a Pod via a ConfigMap and builds upon the [Configure a Pod to Use a ConfigMap](/docs/tasks/configure-pod-container/configure-pod-configmap/) task. 
+
+
+## {{% heading "prerequisites" %}}
+{{< include "task-tutorial-prereqs.md" >}}
+
+## {{% heading "objectives" %}}
+* Update configuration via a ConfigMap mounted as a Volume
+* Update environment variables of a Pod via a ConfigMap 
+
+<!-- lessoncontent -->
+
+## Update configuration via a ConfigMap mounted as a Volume
+
+Use the `kubectl create configmap` command to create a ConfigMap from [literal values](/docs/tasks/configure-pod-container/configure-pod-configmap/#create-configmaps-from-literal-values):
+
+```shell
+kubectl create configmap sport --from-literal=sport=football
+```
+
+[Create a Deployment](/docs/concepts/workloads/controllers/deployment/#creating-a-deployment) manifest named `deployment.yaml` with the below specification. The ConfigMap `sport` is mounted as a Volume on the Pod.
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: configmap-test
+  labels:
+    app: nginx
+spec:
+  replicas: 1
```

Comment:
BTW, it's more realistic to have at least 2 replicas.

Reply:
Acknowledged.

Reply:
Resolved, increased replicas to 3.